package.json

```diff
@@ -13,7 +13,7 @@
   },
   "peerDependencies": {
     "babel-core": "^6.0.0",
-    "webpack": "1 - 2 || 2.0.6-beta || 2.0.7-beta"
+    "webpack": "1 - 2 || 2.0.6-beta || 2.0.7-beta || 2.1.0-beta.0"
```

Comment:
`1 - 2` already matches `2.1.0` anyway, so yes, this may as well just be `^2.1.0-beta.0` as you suggest.

Open a new PR with your suggestion and I'll nuke this one :)